lib/sipper/runner.ex

```diff
@@ -14,6 +15,9 @@ defmodule Sipper.Runner do
     Sipper.FeedParser.parse(feed)
   end
 
+  defp order_by_oldest(episodes, :false), do: episodes
+  defp order_by_oldest(episodes, :true), do: episodes |> Enum.reverse
```

Comment:
You can write `false` instead of `:false` and the same for `true`. (They're implemented as atoms under the hood, so `:true  == true`.)